modules/MIPS/measure_activate.php

```diff
@@ -123,11 +127,19 @@
 	border: 1px solid #333333;
 	border-radius: 4px;
 	color: #333333;
+	background: #66bb6a;
 	display: inline-block;
 	font-family: monospace;
 	margin: 0.25ex 0.15ex 0.15ex 0ex;
 	padding: 0 0.75ex;
 	font-size: 9pt;
+}
+.ma-title {
+	font-size: 20px; 
+	margin-left: 15px;
+}
+.row{
```

Comment:
Bueeeehhh whats this for??? I think bootstrap has a class called `row` which does pretty what you want to accomplish?? why redefining that class ? @punwai

Reply:
@muarachmann This is because rows come with a custom margin-left. I want the bootstrap rows to not have any margins but have the same grid functionality so I have to do this.